src/export/bops/mocks/payload.ts

```diff
@@ -1195,36 +1195,13 @@ export const mockExpectedBOPSPayload = {
         auto_answered: true,
       },
     },
-    {
-      question:
-        "Have you already told us that you are doing works to a tree or hedge?",
-      responses: [
-        {
-          value: "No",
-        },
-      ],
-      metadata: {
-        auto_answered: true,
-      },
-    },
-    {
-      question: "Are there any protected trees on the property?",
-      responses: [
-        {
-          value: "No",
-        },
-      ],
-      metadata: {
-        auto_answered: true,
-      },
-    },
```

Comment:
These two questions are no longer in the up-to-date flow, so we no longer expect them to be in the payload either